database/seed.js

```diff
@@ -0,0 +1,57 @@
+const faker = require('faker');
+const Review = require('../database/index.js');
+
+const hereForArr = ['', 'Breakfast', 'Brunch', 'Lunch', 'Dinner', 'Dessert', 'Coffee or tea', 'Snacks', 'Drinks', 'Late night food', 'Other'];
+const travelerTypeArr = ['Couples', 'Family', 'Friends', 'Business', 'Solo'];
+
+// Stretch goal: Make it so same user can't leave multiple reviews
+// and highest userId equals total number of users minus 1
+
+const seed = () => {
+  // generate data for 100 restaurants
+  for (let i = 0; i < 100; i += 1) {
+    // generate between 0 and 200 reviews for each restaurant
+    const reviewNum = Math.floor(Math.random() * 200);
+    const claimed = faker.random.boolean();
+    const restaurantName = `${faker.random.word()} Restaurant`;
```

Comment:
Love the random.word() + 'Restaurant'.

Reply:
Thanks! I figured it was the simplest way to randomly generate a name that was believably a restaurant.